frontend/components/buttons/Button/_styles.scss

```
@@ -39,7 +39,7 @@ $base-class: 'button';
   letter-spacing: 1px;
   border-radius: 0;
   font-size: $medium;
-  font-family: 'Oxygen', $helvetica;
+  font-family: 'Nunito Sans';
```

```suggestion
  font-family: 'Nunito Sans', sans-serif;
```

`sans-serif` seems to be used as the fallback elsewhere.

---

Great catch. Thanks!